src/main/java/teammates/common/datatransfer/FeedbackMcqQuestionDetails.java

```diff
@@ -171,7 +171,8 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(boolean sessionI
                             Slots.FEEDBACK_RESPONSE_TEXT, Const.ParamsNames.FEEDBACK_RESPONSE_TEXT,
                             Slots.MCQ_PARAM_IS_OTHER_OPTION_ANSWER,
                                     Const.ParamsNames.FEEDBACK_QUESTION_MCQ_ISOTHEROPTIONANSWER,
-                            Slots.MCQ_CHOICE_VALUE, Sanitizer.sanitizeForHtml(existingMcqResponse.getOtherFieldContent()),
+                            Slots.MCQ_CHOICE_VALUE,
+                                    SanitizationHelper.sanitizeForHtml(existingMcqResponse.getOtherFieldContent()),
```

Comment:
I believe the lines were formatted so as to show which slot the value is supposed to take.